src/standalone/Soil/boundary_conditions.jl

```diff
@@ -569,6 +575,39 @@ function WaterHeatBC(; water, heat)
     return WaterHeatBC{typeof(water), typeof(heat)}(water, heat)
 end
 
+"""
+    EnergyFreeDrainage <: AbstractEnergyHydrologyBC
+
+A concrete type of soil boundary condition, for use at
+the BottomBoundary only, where the fluxes are set to be
+`F_liq = -K∇h = -K`, `F_energy = -K ρe_liq`.
+
+That is, this enforces that the free drainage boundary condition
+for liquid water is paired the the corresponding loss of energy
+that that entails.
+"""
+struct EnergyFreeDrainage <: AbstractEnergyHydrologyBC end
```

Comment:
I wonder if `EnergyWaterFreeDrainage` might be a more clear name, to emphasize that this is the BC for both energy and water (not just energy). We could also consider renaming `FreeDrainage` to `WaterFreeDrainage` or having `FreeDrainageWater` and `FreeDrainageWaterEnergy`. I'm not sure what will be best

Reply:
Im also going to open an issue to unify some naming - we have use "WaterHeat" elsewhere, and I think we should change heat to energy. I also wonder about EnergyHydrology as a named and changing that (you've mentioned that before too - it has no "Model" in the title)